### DISCUSSION TITLE
Outlook Calendar Request

### DISCUSSION THREAD

**Discussion by Kalvin23Uni** [PHPMailer/PHPMailer#2573] [Q&A]
Hello,

Are you able to send calendar requests through PHPMailer?

If so, where could I find the documentation on how to do so?

Many thanks,

Kalvin

**Comment by Synchro** [comment]
Create an iCal string however you like, then put it in [`$mail->iCal`](http://phpmailer.github.io/PHPMailer/classes/PHPMailer-PHPMailer-PHPMailer.html#property_Ical). Beware though – how this is handled by clients varies widely, and is horribly inconsistent, so you need to match your format to your audience. Lots more background in [this issue](https://github.com/PHPMailer/PHPMailer/issues/175).
